.github/workflows/test.yml

```diff
@@ -19,8 +19,9 @@ jobs:
   linux:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
I would prefer not wasting CI time because often logs aren't available until all workers finish

Reply:
Good point, let me kick that out.